components/tikv_util/src/time.rs

```diff
@@ -317,12 +350,28 @@ impl Instant {
         }
     }
 
+    pub(crate) fn saturating_elapsed_duration(later: Timespec, earlier: Timespec) -> Duration {
+        if later >= earlier {
+            (later - earlier).to_std().unwrap()
+        } else {
+            error!(
+                "monotonic time jumped back, {:.3} -> {:.3}",
+                earlier.sec as f64 + f64::from(earlier.nsec) / NANOSECONDS_PER_SECOND as f64,
+                later.sec as f64 + f64::from(later.nsec) / NANOSECONDS_PER_SECOND as f64
+            );
+            Duration::from_millis(0)
+        }
+    }
+
     // It is different from `elapsed_duration`, the resolution here is millisecond.
     // The processors in an SMP system do not start all at exactly the same time
     // and therefore the timer registers are typically running at an offset.
     // Use millisecond resolution for ignoring the error.
     // See more: https://linux.die.net/man/2/clock_gettime
-    fn elapsed_duration_coarse(later: Timespec, earlier: Timespec) -> Duration {
+    pub(crate) fn saturating_elapsed_duration_coarse(
```

Comment:
I don't think the rename makes sense as there is no accurate time when using coarse timer, so there is no such "non-saturating` version at all.

Reply:
make sense.